sampler.go

```diff
@@ -406,10 +410,9 @@ func NewRemotelyControlledSampler(
 	sampler := &RemotelyControlledSampler{
 		samplerOptions: options,
 		serviceName:    serviceName,
-		timer:          time.NewTicker(options.samplingRefreshInterval),
 		manager:        &httpSamplingManager{serverURL: options.samplingServerURL},
+		doneChan:       make(chan *sync.WaitGroup),
```

Comment:
Interesting. Most other times we've done this, we created a empty chan (chan struct{}) and closed it inside Close() and had the pollControllerWithTicker intercept the closing of the chan and then calling s.wg.Done() vs sending the actual wg inside chan to the go routine. Both work but I've not seen it done this way anywhere inside our codebase.